### PR TITLE
Revised mfe_setup

### PR DESCRIPTION
**Pull request by fredgent**
Revised mfe_setup for positive pressure/density and added path to plotting calls following removal of imports in `pysac/mhs_atmosphere/__init__.py`

### CODE REVIEW

**Comment by Cadair** [comment]
You want to know the lovely thing about this? If I merge this and the rerun my Thesis, all the plots and my table will update automatically :dancers:

**Comment by fredgent** [comment]
Haha, cool

File: pysac/mhs_atmosphere/parameters/model_pars.py [hunk 1]
```diff
@@ -22,16 +22,16 @@
...
              'model': 'mfe_setup',
              'phratio': 0.0*u.one,
              'pixel': 0.36562475*u.Mm,  #(HMI pixel)
-             'radial_scale': 0.044*u.Mm,
+             'radial_scale': 0.14*u.Mm,
              'nftubes': 1,
-             'B_corona': 4.35e-4*u.T,
-             'pBplus': 4.250e-4*u.T}
+             'B_corona': 4.85e-4*u.T,
+             'pBplus': 0.0e-4*u.T}
```
Comment (Cadair) [inline pysac/mhs_atmosphere/parameters/model_pars.py]: So this removes the background field??

**Comment by fredgent** [comment]
If that is so, just double check b_01, and b_02. These should be swapped
in the version you sent me the other day.

File: pysac/mhs_atmosphere/parameters/model_pars.py (hunk 1 above)
Comment (fredgent) [inline pysac/mhs_atmosphere/parameters/model_pars.py]: No the background field is

'B_corona'

It removes the enhancement to the pressure, which was introduced for the
corona in Paper41, but was never applied by Viktor

'pBplus'

Comment (Cadair) [inline pysac/mhs_atmosphere/parameters/model_pars.py]: oh right ok.

**Comment by Cadair** [comment]
Also, this seems to have made the FWHM of the footpoint a fair bit larger. Have you compared this, or shall I do it?

File: pysac/mhs_atmosphere/parameters/model_pars.py (hunk 1 above)
Comment (fredgent) [inline pysac/mhs_atmosphere/parameters/model_pars.py]: In fact for absolute clarity mfe_setup now has

B_0z = exp(-z/0.4Mm)
B_z = 128.5mT * B_0z^2 * G + 0.485mT

Fred

**Comment by fredgent** [comment]
I haven't, but in fact as it stands, if I try to reduce f_0, then the
pressure or density becomes negative, so this is close to the lower
limit. I can also make f_0 less than 0.01, but that make the flux tube
very thin and I am sure Viktor said his flux tube was at least 100km in
diameter.

Let me know, as I want to avoid any glaring discrepancies.

Fred

**Comment by fredgent** [comment]
Also, I notice from the 2D plot, that there are patches of p-beta <1 at
the edge around z=0.8Mm. Is this consistent with your numerical atmosphere?
This is away from the main flux tube structure and will have been
outside your flux surface analysis, but I would be interested to know if
there are any visible effects on the velocity field in you movies of the
whole domain.

**Pull request merged**